<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layCentral">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="1" column="1">
      <widget class="QScrollArea" name="scrAreaInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>147</width>
          <height>179</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Результат:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="teResult">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="bGetResults">
       <property name="text">
        <string>Получить результат</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bSaveToFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Записать результат в файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>291</width>
      <height>678</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="layScroll">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Эритроциты (RBC):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="sbRBC">
       <property name="suffix">
        <string> трл/л</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Гемоглобин (HGB):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="sbHGB">
       <property name="suffix">
        <string> г/л</string>
       </property>
       <property name="maximum">
        <double>1000.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Средний объём
 эритроцитов (MCV):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="sbMCV">
       <property name="suffix">
        <string> фл</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Среднее содержание
 гемоглобина
 в эритроците (MCH):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="sbMCH">
       <property name="suffix">
        <string>  пг</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Средняя концентрация
 гемоглобина
 в эритроците (MCHC):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="sbMCHC">
       <property name="suffix">
        <string> г/дл</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Цветовой показатель 
 (ЦП):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="sbCF"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ретикулоциты (RTC):</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="sbRTC">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Тромбоциты (PLT):</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="sbPLT">
       <property name="suffix">
        <string> млн/л</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Лейкоциты (WBC):</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="sbWBC">
       <property name="suffix">
        <string> млн/л</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Сегментоядерные
 нейтрофилы (NE%):</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="sbNE">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Палочкоядерные
 нейтрофилы:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="sbPN">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Миелоциты (Mie):</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="sbMie">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Метамиелоциты
 (юные):</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="sbMieY">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Лимфоциты (LYM%):</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QDoubleSpinBox" name="sbLYM">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Моноциты (MON%):</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QDoubleSpinBox" name="sbMON">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Эозинофилы (EO%):</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QDoubleSpinBox" name="sbEO">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Базофилы (BA%):</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QDoubleSpinBox" name="sbBA">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Атипичные
 мононуклеары
 (вироциты):</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QDoubleSpinBox" name="sbVir">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Плазматические
 клетки
 (плазмоциты):</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QDoubleSpinBox" name="sbPlaz">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>СОЭ (ESR):</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QDoubleSpinBox" name="sbESR">
       <property name="suffix">
        <string> мм/час</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Гематокрит (HCT):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="sbHCT">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Пол (возраст):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbGender">
       <item>
        <property name="text">
         <string>Муж.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Женск.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Реб. 13-15л.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Реб. 7-12 л.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Реб. 1-6 л.</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="scrWidget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
